### PR TITLE
Rewrite test that won't break after I start checking for left vs righ…

### PR DESCRIPTION
**Pull request by graceolivia**
…t side increases:

Added support for right-side increases and tests for those increases. I added this functionality to `makeMiddleRowStitchCountChange` and added tests under the class `FullChartPrinterTests` and to a new class `RightIncreaseMiddlePrinterTests`

### CODE REVIEW

File: Tests/SwiftCliCoreTests/ChartConstructorTest.swift [hunk 1]
```diff
@@ -105,6 +105,21 @@ class RightDecreaseMiddlePrinterTests: XCTestCase {
...
 
+class RightIncreaseMiddlePrinterTests: XCTestCase {
+    func testOneRightIncrease() throws {
+        let result = ChartConstructor().makeMiddleRowStitchCountChange(width: 5, rDiff: 1, lDiff: 0)
+        let expectedOutput = "├─┼─┼─┼─┼─┘\n"
+        expect(result).to(equal(expectedOutput))
+
+    }
+
+    func testOneMultiIncrease() throws {
```
Comment (hnlee) [inline Tests/SwiftCliCoreTests/ChartConstructorTest.swift:116]: Do you mean `testMultipleRightIncrease`?

**Review by hnlee — CHANGES_REQUESTED** [review]

File: Tests/SwiftCliCoreTests/ChartConstructorTest.swift (hunk 1 above)
Comment (graceolivia) [reply Tests/SwiftCliCoreTests/ChartConstructorTest.swift:116]: Yes!

File: Sources/SwiftCliCore/ChartConstructor.swift [hunk 2]
```diff
@@ -81,6 +82,11 @@ public class ChartConstructor {
             rightStitches = "─┼─┐\n"
         case (rDiff == 0):
             rightStitches = "─┤\n"
+        case (rDiff == 1):
+            rightStitches = "─┼─┘\n"
+        case (rDiff > 1):
+            let midRightSt = String(repeating: "─┴", count: ((abs(rDiff))-1))
+            rightStitches = "─┼\(midRightSt)─┘\n"
```
[the remark this answers is not in the document]
Comment (graceolivia) [reply Sources/SwiftCliCore/ChartConstructor.swift:89]: Fixed!

**Review by hnlee — APPROVED** [review]
Great work, you're on a roll here!

**Pull request merged**